application/clicommands/CheckCommand.php

```diff
@@ -120,8 +120,8 @@ public function hostAction()
             }
 
             $now = new \DateTime();
```

Comment:
Is there already a PR that fixes the imports?

Reply:
Nope!